docs/src/components/Example.js

```diff
@@ -14,6 +16,7 @@ type Props = {|
   name: string,
   showHeading?: boolean,
   showCode?: boolean,
+  skipContrastCheck?: boolean,
```

Comment:
Temporary fix until docs are converted to new components - not documenting to prevent use in the short term